src/js/components/Box/StyledBox.js

```diff
@@ -73,6 +73,9 @@ const directionStyle = (direction, theme) => {
   ];
   if (direction === 'row-responsive' && theme.box.responsiveBreakpoint) {
     const breakpoint = theme.global.breakpoints[theme.box.responsiveBreakpoint];
+    if (!breakpoint.edgeSize) breakpoint.edgeSize = theme.global.edgeSize;
+    if (!breakpoint.borderSize) breakpoint.borderSize = theme.global.borderSize;
+    if (!breakpoint.size) breakpoint.size = theme.global.size;
```

Comment:
@jcfilben It seems like we are repeating this logic in multiple places which may or may not be we want, but it at least gives me enough pause to wonder if there might be an opportunity to handle this uniformly across the project in a utils function perhaps?

You also mentioned in your write up needing to investigate Layer. I peeked and yes it does look like in addition to StyledBox.js and border.js (which you have identified), StyledLayer.js and StyledHeading.js will need to be updated, as well as possibly responsive.js and styles.js which might be tangentially related.

Did you happen to explore any solutions that might be applied more universally? 

I haven't vetted these yet, but perhaps an approach which instead of calling 
```
const breakpoint =
    theme.box.responsiveBreakpoint &&
    theme.global.breakpoints[theme.box.responsiveBreakpoint];
```
we could instead call a utility function which returns a normalized breakpoint style object?
```
const breakpoint =
    theme.box.responsiveBreakpoint &&
    getBreakpointStyle(theme.box.responsiveBreakpoint);
```

The function might live in `utils/responsive.js` and would take in a t-shirt size string as a parameter and would return an object in the shape of:
```
{
  borderSize: { ... },
  edgeSize: { ... },
  size: { ... },
  value: ...
}
```

Again, haven't vetted this approach, but curious if you have explored or would be willing to explore something along these lines?

@ShimiSun or @ericsoderberghp might have some thoughts before pursuing this angle.

Reply:
I agree that this would be a solution worth exploring because it could solve the duplicated code issue as well as solving the breakpoint issue in other components. I will explore this option and update this PR as I go